# MotorControl.srv
float32 speed
---
bool success
